skbio/sequence/_dna.py

```diff
@@ -132,6 +142,35 @@ def degenerate_map(cls):
             "H": set("ACT"), "V": set("ACG"), "N": set("ACGT")
         }
 
+    @classproperty
+    def default_gap_char(cls):
+        """Gap character to use when constructing a new gapped sequence.
+
+        This character is used when it is necessary to represent gap characters
+        in a new sequence. For example, a majority consensus sequence will use
+        this character to represent gaps.
+
+        Returns
+        -------
+        str
+            Default gap character.
+
+        """
+        return '-'
+
+    @classproperty
+    @stable(as_of='0.4.0')
```

Comment:
Can you remove this API stability decorator, as well as the API decorators for `complement_map`, `nondegenerate_chars`, and `degenerate_map`? I just noticed that the API stability description is being duplicated in the final docstring.